python/QC4_HV.py

```diff
@@ -0,0 +1,49 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand,envCheck
+from options import parser
+import os
+import xlrd
+
+args = parser.parse_args()
+envCheck('GEM_BASE')
+dataPath  = os.getenv('GEM_BASE')
+cmd = ["python","%s/python/Produce_Config_File.py"%(dataPath)]
+
+for filelist in args.file:
+	cmd.append(filelist)
+	pass
+
+cmd.append("--CanvTitleX=Divider Current #left(#muA#right)")
+cmd.append("--CanvTitleY=Applied Voltage #left(kV#right)")
+cmd.append("--SelectColumnX= 3")
+cmd.append("--SelectColumnY= 1")
+cmd.append("--SelectRowStart= 3")
+cmd.append("--SelectRowEnd= 37")
+cmd.append("--CanvRangeX= 0,1000")
+cmd.append("--CanvRangeY= 0,7")
```

Comment:
Will these options be properly understood with a space character after the equals sign?